notebook/static/notebook/js/notebook.js

```diff
@@ -1944,6 +1948,7 @@ define([
     Notebook.prototype.set_insert_image_enabled = function(enabled) {
         if (enabled) {
             $('#insert_image').removeClass('disabled');
+            $('.image_link').removeAttr("aria-disabled");
         } else {
             $('#insert_image').addClass('disabled');
```

Comment:
I have made the necessary changes(changed the attribute value to true if they are disabled again) however, instead of adding and deleting the aria-disabled attribute I used the attribute value(true/false) to keep track of the state. https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled

Reply:
Thanks, switching it to true and false is good.